File: ble-common/src/main/java/no/nordicsemi/android/ble/common/profile/bp/BloodPressureUnit.java
package no.nordicsemi.android.ble.common.profile.bp;

import android.support.annotation.IntDef;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

@SuppressWarnings("WeakerAccess")
@Retention(RetentionPolicy.SOURCE)
@IntDef(value = {
		BloodPressureTypes.UNIT_mmHg,
		BloodPressureTypes.UNIT_kPa
})
public @interface BloodPressureUnit {}

File: ble-common/src/main/java/no/nordicsemi/android/ble/common/profile/bp/BloodPressureUnit.kt
package no.nordicsemi.android.ble.common.profile.bp

import androidx.annotation.IntDef

import java.lang.annotation.Retention
import java.lang.annotation.RetentionPolicy

@Retention(RetentionPolicy.SOURCE)
@IntDef(value = [BloodPressureTypes.UNIT_mmHg, BloodPressureTypes.UNIT_kPa])
annotation class BloodPressureUnit